frontend/src/component/Dashboard/RecentActivity.styl

```diff
@@ -26,11 +29,11 @@
     &.read
       font-weight normal
     &:hover
-      background-color fourthColor
-    &:nth-child(even)
-      background-color grey-hover
-      &:hover
-        background-color fourthColor
+      color off-white
+    &:nth-child(even):not(:hover)
+      background-color lightGrey
+      // &:hover
```

Comment:
delete comment
you can let comments when you are not 100% sure of the change. This change is safe, we wont ever again use variables like fourthColor